src/cc_toolchain/wrapper/k8_gcc-k8_gcc

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash --norc
+
+# Copyright 2018 Josh Pieper, jjp@pobox.com.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exec /usr/bin/x86_64-linux-gnu-gcc -Wl,--no-as-needed "$@"
```

Comment:
@garethellis0 can you do better than this?

Reply:
Ok, I don't immediately see a better way :cry:. I'm fine with this as long as we leave a comment and open an issue with https://github.com/bazelbuild/bazel asking if this is the correct behavior. I can do make the issue or you can, just lemme know!

Reply:
drafting an email to bazel-discuss@googlegroups.com according to this message: 
> ATTENTION! Please read and follow:
> - if this is a _question_ about how to build / test / query / deploy using Bazel, or a _discussion starter_, send it to bazel-discuss@googlegroups.com
> - if this is a _bug_ or _feature request_, fill the form below as best as you can.